disk-buffering/src/main/java/io/opentelemetry/contrib/disk/buffering/exporters/SignalStorageExporter.java

```diff
@@ -13,11 +15,10 @@
 import java.util.Collection;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
 /** Internal utility for common export to disk operations across all exporters. */
-final class SignalStorageExporter<T> {
+class SignalStorageExporter<T> {
```

Comment:
Keep `final` for package private as well.
```suggestion
final class SignalStorageExporter<T> {
```